### PR TITLE
Bumped recipe version to 1.10.5

### PR DESCRIPTION
**Pull request by jlstevens**
This bumps the recipe version up for the 1.10.5 bug fix release.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,10 +25,11 @@ requirements:
     - param >=1.6.1,<2.0
     - numpy >=1.0
     - matplotlib >=2.1
-    - bokeh >=0.12.15
+    - bokeh >=0.12.15,<=0.12.16
```
Comment (jakirkham) [inline recipe/meta.yaml]: Should this be added upstream as well? Also guessing this should be `<0.12.16` or are we ok with `0.12.16` and expect a break in `0.12.17`?

ref: https://github.com/ioam/holoviews/blob/v1.10.5/setup.py#L20

File: recipe/meta.yaml [hunk 2]
```diff
@@ -25,10 +25,11 @@ requirements:
...
     - jupyter
     - notebook
     - ipython >=5.4.0
```
Comment (jakirkham) [inline recipe/meta.yaml]: We may also consider pushing this constraint upstream as well (not that it changed in this PR).

ref: https://github.com/ioam/holoviews/blob/v1.10.5/setup.py#L17

**Comment by jakirkham** [comment]
Couple questions about dependencies. Otherwise LGTM.

File: recipe/meta.yaml (hunk 1 above)
Comment (philippjfr) [reply recipe/meta.yaml]: We are actually compatible with 0.13.0 which is due to be released next week. It is pinned in the build uploaded to the pyviz channel which was built from this recipe: https://github.com/ioam/holoviews/blob/master/conda.recipe/meta.yaml#L28

Therefore we'll likely have another release shortly after the bokeh release to fix that.

File: recipe/meta.yaml (hunk 2 above)
Comment (jlstevens) [reply recipe/meta.yaml]: That constraint is expressed in the [``meta.yaml``](https://github.com/ioam/holoviews/blob/master/conda.recipe/meta.yaml) which I used to update this recipe with ``conda render``. I agree we should update ``setup.py`` too so I'll make a holoviews PR for that.

Comment (philippjfr) [reply recipe/meta.yaml]: Do we really need to update setup.py to include this constraint now if we want to remove it as soon as next week?

Comment (jlstevens) [reply recipe/meta.yaml]: I was thinking of adding the updated constraint next week so it is ready in time for the next conda-forge recipe update.

Comment (philippjfr) [reply recipe/meta.yaml]: Okay makes sense, that would be ``bokeh >=0.12.15,<=0.13.0``?

Comment (jlstevens) [reply recipe/meta.yaml]: I assume so, yes.

**Comment by jlstevens** [comment]
@jakirkham Hopefully we have answered your questions about the dependency pinning in the comments above.

**Comment by jlstevens** [comment]
Going ahead and merging.

**Pull request merged**